spec/abstractops.html

```diff
@@ -571,19 +571,9 @@ <h1>ToRelativeTemporalObject ( _options_ )</h1>
         1. Let _calendar_ be ? GetOptionalTemporalCalendar(_value_).
         1. Let _fieldNames_ be ? CalendarFields(_calendar_, « *"day"*, *"month"*, *"year"* »).
         1. Let _fields_ be ? ToTemporalDateTimeFields(_value_, _fieldNames_).
-        1. Let _temporalDate_ be ? DateFromFields(_calendar_, _fields_, %Temporal.PlainDate%).
-        1. Let _timeResult_ be ? ToTemporalTimeRecord(_fields_).
-        1. Let _result_ be the new Record {
-            [[Year]]: _temporalDate_.[[ISOYear]],
-            [[Month]]: _temporalDate_.[[ISOMonth]],
-            [[Day]]: _temporalDate_.[[ISODay]],
-            [[Hour]]: _timeResult_.[[Hour]],
-            [[Minute]]: _timeResult_.[[Minute]],
-            [[Second]]: _timeResult_.[[Second]],
-            [[Millisecond]]: _timeResult_.[[Millisecond]],
-            [[Microsecond]]: _timeResult_.[[Microsecond]],
-            [[Nanosecond]]: _timeResult_.[[Nanosecond]]
-          }.
+        1. Let _options_ be ? OrdinaryObjectCreate(%Object.prototype%).
```

Comment:
can this throw? I wouldn’t expect so.
```suggestion
        1. Let _options_ be ! OrdinaryObjectCreate(%Object.prototype%).
```